lisa/sut_orchestrator/libvirt/schema.py

```diff
@@ -133,10 +133,16 @@ def __post_init__(self) -> None:
             self.disk_img_format = DiskImageFormat.QCOW2.value
 
 
+@dataclass_json()
+@dataclass
+class KernelSchema:
+    path: str = ""
+    is_remote_path: bool = False
+
+
 @dataclass_json()
 @dataclass
 class CloudHypervisorNodeSchema(BaseLibvirtNodeSchema):
-    # Local path to the cloud-hypervisor firmware.
-    # Can be obatained from:
-    # https://github.com/cloud-hypervisor/rust-hypervisor-firmware
+    # Local or remote path to the cloud-hypervisor kernel.
     firmware: str = ""
+    kernel: Optional[KernelSchema] = None
```

Comment:
```suggestion
    # DEPRECATED: use the 'kernel' field instead.
    # Local path to the cloud-hypervisor firmware.
    # Can be obtained from:
    # https://github.com/cloud-hypervisor/rust-hypervisor-firmware
    firmware: str = ""
    # Local or remote path to the cloud-hypervisor kernel.
    kernel: Optional[KernelSchema] = None
```

Reply:
Thanks, this gives a better understanding of what we're doing.